src/pages/Home/index.js

```diff
@@ -0,0 +1,534 @@
+                    />
+                    <span className={classes.downloadBtn}>Download Meraki</span>
+                  </Button>
+                </Grid>
+              </Grid>
+            </div>
+          </Container>
+
+          {/* ********************** Section 2 ******************* */}
+
+          <Container sx={{ mt: 4, display: { xs: "none", md: "flex" } }}>
+            <Grid container>
+              <Grid container item xs={4} justifyContent="flex-start">
+                <Card className={classes.typingPopupCard}>
+                  <CardContent>
+                    <Typography color="text.secondary">
+                      I want to be a typing assitant
+                    </Typography>
+                  </CardContent>
+                </Card>
+              </Grid>
+              <Grid item xs={4}></Grid>
+              <Grid container item xs={4} justifyContent="flex-end">
+                <Card className={classes.engineerPopupCard}>
+                  <Typography align="right" color="text.secondary">
+                    I want to be the first software engineer in my family
+                  </Typography>
+                </Card>
+              </Grid>
+            </Grid>
+          </Container>
+          <Container
+            sx={{ mt: 3, display: isActive || isActiveIpad ? "none" : "flex" }}
+          >
+            <Grid container alignItems="center">
+              <Grid item xs={12}>
+                <img src={require("./assets/graphic.svg")} />
+              </Grid>
+            </Grid>
+          </Container>
+
+          {/* ********************** Section 3 ******************* */}
+
+          <Container sx={{ mt: 10 }} maxWidth="sm">
+            <Typography
+              variant="h5"
+              component="h6"
+              align="center"
+              color="textPrimary"
+              gutterBottom
+            >
+              Explore the Learning Tracks
+            </Typography>
+          </Container>
+
+          <Container className={classes.cardGrid} maxWidth="lg">
+            <Grid container spacing={4}>
+              <Grid item xs={12} ms={6} md={4}>
+                <Card className={classes.card}>
+                  <CardMedia
+                    component="img"
+                    image={require("./assets/python.svg")}
+                    alt="green iguana"
+                  />
+                  <CardContent className={classes.cardConten}>
+                    <Typography
+                      gutterBottom
+                      variant="h6"
+                      align="center"
+                      component="div"
+                    >
+                      Python
+                    </Typography>
+                    <Typography align="center">
+                      Get familiar with programming with bite sized lessons
+                    </Typography>
+                  </CardContent>
+                </Card>
```

Comment:
I'd also recommend making these Components as well so it'd be something like this:
```
function PathwayCard(props) {
  const classes = useStyles();
  return (
    <Card className={classes.card}>
      <CardMedia
        component="img"
        src={require("./assets/" + props.imgSrc + ".svg")}
        alt={props.alt || "image for " + props.trackName}
      />
      <CardContent className={classes.cardContent}>
        <Typography gutterBottom variant="h6" align="center" component="div">
          {props.trackName}
        </Typography>
        <Typography align="center">{props.description}</Typography>
      </CardContent>
    </Card>
  );
}
```
And then you could make some sort of `PathwayTable` component rendering an Array of `PathwayCard` components.  But I'm assuming this breaking up into components will happen naturally once you starting pulling this information from the database.  (See also: https://reactjs.org/docs/thinking-in-react.html#step-1-break-the-ui-into-a-component-hierarchy.)

Reply:
Ok sure